app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/cases_list.scala.html

```diff
@@ -0,0 +1,53 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.models.Case
+@import uk.gov.hmrc.tariffclassificationfrontend.utils.Dates
+@(cases: Seq[Case])
+
+<table class="govuk-table">
+    <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+            <th class="govuk-table__header" scope="col">Reference</th>
+            <th class="govuk-table__header" scope="col">Description</th>
+            <th class="govuk-table__header" scope="col">Trader</th>
+            <th class="govuk-table__header" scope="col">Date Received</th>
+            <th class="govuk-table__header" scope="col">Status</th>
+            <th class="govuk-table__header" scope="col">Type</th>
+            <th class="govuk-table__header" scope="col">Days Elapsed</th>
+        </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+        @for(c <- cases) {
+            <tr class="govuk-table__row">
+                <th class="govuk-table__header" scope="row">
+                    <a href="#">@c.reference</a>
+                </th>
+                <td class="govuk-table__cell">@c.description</td>
+                <td class="govuk-table__cell">@c.traderName</td>
+                <td class="govuk-table__cell">@Dates.format(c.createdDate)</td>
+                <td class="govuk-table__cell">@c.status</td>
+                <td class="govuk-table__cell">@c.applicationType</td>
+                <td class="govuk-table__cell">@c.daysElapsed</td>
+            </tr>
+        }
+    </tbody>
+</table>
+
+@if(cases.isEmpty) {
+    <p>There's no cases in this queue yet</p>
```

Comment:
The correct English grammar I believe is `There are no ...`

Reply:
Sorted although I don't think it matters for this case